configure

```diff
@@ -1885,6 +1885,12 @@ EOF
                 ARCH="${ARCH}+dfltcc-inflate"
             fi
 
+            if test $builddfltccinflate -eq 1 -o $builddfltccdeflate -eq 1; then
+                ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} dfltcc_common.o"
+                ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} dfltcc_common.lo"
+                ARCH="${ARCH}+dfltcc-common"
```

Comment:
**question:** 
Is this actually something that needs to be added to the ARCH string?

Reply:
I don't remember why I added them and I won't object to removing them - they seem to only cause confusion.

Reply:
There are other architectures that require adding suffixes to `ARCH` and won't work without the addition. It seems to be specific to compiler version. Some compiler versions enable extensions by default, others enable them only when FP or vector unit is explicitly enabled on command line.

Reply:
I see now why they are needed elsewhere, thanks. In this particular case they definitely don't serve this function. So let's drop them (perhaps in a separate PR?).